javascript/house-style.md

```diff
@@ -3,6 +3,7 @@
 This document outlines the way we write JavaScript. It's a living styleguide – it will grow as our practices do.
 
 - [General Principles](#general-principles)
+  - [Progressively Enhance](#progressively-enhance)
```

Comment:
Could this heading be in sentence case please? 

It's not made explicit in the language style guide and this document is a horrible mix of title case and sentence case, but sentence case is more common across the repo overall